sky/resources.py

```diff
@@ -553,7 +553,8 @@ def _validate_and_set_region_zone(self, region: Optional[str],
             # Try to infer the cloud from region/zone, if unique. If 0 or >1
             # cloud corresponds to region/zone, errors out.
             valid_clouds = []
-            enabled_clouds = global_user_state.get_enabled_clouds()
+            enabled_clouds = sky_check.get_enabled_clouds(
+                raise_if_no_cloud_access=True)
```

Comment:
Can we update the docstr of the function to include the `Raises`?

Reply:
We already have that?

<img width="973" alt="image" src="https://github.com/skypilot-org/skypilot/assets/74357442/89438d06-37bb-485b-b54f-47ba920eed04">

Reply:
I mean the current caller function in this file :)